tests/fps_tests.py

```diff
@@ -704,6 +706,40 @@ def test_expected_esld(self):
         self.assertEqual(loaded_sets, expected_sets)
         self.assertEqual(fp.error_list, [])
 
+class TestFindDiff(unittest.TestCase):
+    def test_basic_find_diff(self):
+        old_json_dict = {
+            "sets":
+            [
+                {
+                    "primary": "https://primary.com",
+                    "ccTLDs": {
+                        "https://primary.com": ["https://primary.ca"]
+                    }
+                }
+            ]
+        }
+        new_json_dict = {
+            "sets":
+            [
+                {
+                    "primary": "https://primary2.com",
+                    "ccTLDs": {
+                        "https://primary2.com": ["https://primary2.ca"]
+                    }
+                }
+            ]
+        }
+        old_fp = FpsCheck(fps_sites=old_json_dict,
+                     etlds=None,
+                     icanns=set(["ca"]))
+        new_fp = FpsCheck(fps_sites=new_json_dict,
+                          etlds=None,
+                          icanns=["ca"])
```

Comment:
I think it would be much clearer if we didn't rely on `FpsCheck` in this unit test, since `find_diff_sets` could easily be tested without specifying all the metadata for FPSes.

I also think it would be clearer if we spelled out the inputs and outputs directly, rather than computing them (via `old_fp.load_sets()` and `new_fp.load_sets()`). See e.g. https://testing.googleblog.com/2014/07/testing-on-toilet-dont-put-logic-in.html.